apps/frontend/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 	<head>
 		<meta charset="UTF-8" />
-		<link rel="icon" type="image/svg+xml" href="/vite.svg" />
+		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
 		<link rel="stylesheet" href="./src/assets/css/styles.css" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>git-fit</title>
```

Comment:
Let's update this too 
```suggestion
		<title>GitFit</title>
```

Reply:
I was thinking about that ;)